manifests/e/EdrawSoft/EdrawMax/CN/15.0.0.1399/EdrawSoft.EdrawMax.CN.installer.yaml

```
@@ -0,0 +1,20 @@
+# Created with YamlCreate.ps1 Dumplings Mod
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.10.0.schema.json
+
+PackageIdentifier: EdrawSoft.EdrawMax.CN
+PackageVersion: 15.0.0.1399
+InstallerType: inno
+Scope: machine
+UpgradeBehavior: install
+FileExtensions:
+- eddx
+- edt
+- edx
+ProductCode: '{037BAB81-3DF7-4381-A72C-A26B57C03548}_is1'
+ReleaseDate: 2025-10-23
+Installers:
+- Architecture: x64
+  InstallerUrl: https://cc-download.wondershare.cc/cbs_down/edraw-max_cn_64bit_15.0.0_full5374.exe
```

Hi @SpecterShell,

Is there a good `PackateUrl` for this (edit) application? An Internet search led to [this page](https://www.edrawsoft.cn/edrawmax/), which had a slightly different URL:

```suggestion
  InstallerUrl: https://cc-download.edrawsoft.cn/edraw-max_cn_full5374.exe
```

Should this be used instead?

---

Nope. This is a bootstrapper that downloads and runs the real installer.
Also, EdrawSoft has been part of Wondershare for a long while.